### PR TITLE
v1 - Basic commands and API functionality

### PR DESCRIPTION
**Pull request by tm21cy**
**v1 is a Minimum Viable Product - it should not be used for deployment purposes in a stable production environment.**

Commits:
https://github.com/tm21cy/Bit/commit/206098291242477102d83e1e13c721b7aabab62f - Code quality improvements centered around the `commands` folder.
https://github.com/tm21cy/Bit/commit/a7230fdc1910095b2a32e5bf004d6cf81f9ecdc2 - Introduction of API routing; docs to be added later. Adds basic POST/GET routes for Profiles, Notifications, LikeUsers, and Comments. PATCH requests to be added in subsequent PRs.
https://github.com/tm21cy/Bit/commit/aa1ee4885b881d2fae7ee8b827206e5917efbc64 - Code quality improvements centered around the `events` folder.
https://github.com/tm21cy/Bit/commit/7f45d2ae73cd7371f4bd064b95b237e55d9eb68a - Adds an enumerator object for processing the `badge_flags` field in the `profiles` table.
https://github.com/tm21cy/Bit/commit/a23456ce7c2e4be656d1de472f22fec8708df7f7 - Adds ADTs for typing response objects.
https://github.com/tm21cy/Bit/commit/291f32023c418c38fd856a2eefb2fb3505c7afea - Adds functionality for basic profile generation. Includes a badge handler (via `badges` and `badgesArray`), as well as a Menu component render tool and profile Embed render tool.
https://github.com/tm21cy/Bit/commit/016248e53e580e8db5115180dca9677349a924e8 - Code quality improvements centered around the `services` folder.
https://github.com/tm21cy/Bit/commit/6e1f7829a52e92b661add8d426e5e3db9c45b7bc - Adds a security service to prevent unsafe passwords & dangerous eval access.
https://github.com/tm21cy/Bit/commit/91df61c2e7880dc6997c90f12ca5850e8741afa6 - Code quality improvements centered around the `utilities/general` class.
https://github.com/tm21cy/Bit/commit/af252f174c063ff8f56aa98735a230d2fdc2ac45 - Adds a textCommand for eval functions.
https://github.com/tm21cy/Bit/commit/be3c1f121ddbbdfed8667eed40d8275cfabb8b9d - Necessary updates to metadata files.

Upcoming changes:
- Comments and documentation
- Deeper profile functionality
- Orthogonality in class methods to allow for abstract implementation

### CODE REVIEW

File: src/routes/Comments.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  }
+
+  /**
+   * GET request that returns a truncated list of the 5 most recent comments made targeting the user found with `targetID`.
+   * @param targetID The Discord ID snowflake of the targeted user.
+   * @returns Promise containing a Get object, which contains the comments as a Comment array, and a status code.
+   */
+  async retrieveComments(targetID: string): Promise<Get> {
+    const retTuple = (await db.query(
+      `SELECT * FROM comments WHERE target_id = '413462464022446084' ORDER BY id DESC`,
```
Comment (avasilic) [inline src/routes/Comments.ts]: Is the target ID as `413462464022446084` here on purpose?
```suggestion
      `SELECT * FROM comments WHERE target_id = '${targetID}' ORDER BY id DESC`,
```

File: src/services/security.ts [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+    }
+
+    const headers = {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${process.env.EVAL_CHECK_API_KEY}`,
+    };
+
+    return await axios
+      .get(
+        `${process.env.EVAL_CHECK_URL_BASE}/${process.env.EVAL_CHECK_URL_PATH}/${process.env.CLIENT_ID}`,
```
Comment (avasilic) [inline src/services/security.ts]: The aforementioned API does not work. Stay tuned for suggested changes in the eval command.

File: src/utilities/profiles/render.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  let user_id = profile.user_id;
+  let bio = profile.bio;
+  let display_picture = profile.display_picture ?? process.env.CLIENT_PFP;
+  let hits = profile.hits;
+  let likes = profile.likes;
+  let id = profile.id;
+  let muted = profile.muted;
+
+  // Preliminary data parsing
+  let embedColor = "Blurple";
```
Comment (avasilic) [inline src/utilities/profiles/render.ts]: Prefer Hello World! theme.
```suggestion
  let embedColor = "#5856d6";
```

**Review by avasilic — CHANGES_REQUESTED** [review]
Please review the comments. I will fix Eval.

**Comment by tm21cy** [comment]
LGTM

**Review by avasilic — APPROVED** [review]

**Pull request merged**